### ISSUE TITLE
outliers_dmst_cde validation

### ISSUE THREAD

**Issue by DavorJ** [DOV-Vlaanderen/groundwater-logger-validation#7]
We wanted to evaluate the DRME_DMST_CDE field taken from the database. 

[Here](https://github.com/DOV-Vlaanderen/groundwater-logger-validation/blob/outliers/src/r/outliers/outliers_dmst_cde.pdf) are the results.

The _DEL_ and _VLD_ category seem to reference the duplicates in data:

![image](https://user-images.githubusercontent.com/4161918/58820970-4d036400-8634-11e9-93d0-a274f4ac04aa.png)
![image](https://user-images.githubusercontent.com/4161918/58820984-555b9f00-8634-11e9-876c-34cfc475d9bf.png)

The "DUPES=..." annotation is calculated based on duplicate timestamps. They match perfectly.

The _INV_ seems to reference wrong data, but is inconsistent. Here some examples:
![image](https://user-images.githubusercontent.com/4161918/58821213-dfa40300-8634-11e9-8ef7-7d80c50bb453.png)
Why not the black point between the red points? And the two first red points do not seem to be outliers, given the others.

![image](https://user-images.githubusercontent.com/4161918/58821395-45908a80-8635-11e9-8ac4-65396bc5bab5.png)
Clearly outliers, but not flagged.

![image](https://user-images.githubusercontent.com/4161918/58821453-62c55900-8635-11e9-9fa9-def4edfd6db9.png)
What is wrong with these?

![image](https://user-images.githubusercontent.com/4161918/58821494-7bce0a00-8635-11e9-9a00-180122f8380b.png)
Inconsistent.

![image](https://user-images.githubusercontent.com/4161918/58821618-c18ad280-8635-11e9-945d-4e6006d1dc37.png)
The second spike = OK, but the first?

![image](https://user-images.githubusercontent.com/4161918/58821745-03b41400-8636-11e9-878b-2bf3abb411e0.png)
Inconsistent.

![image](https://user-images.githubusercontent.com/4161918/58821790-20e8e280-8636-11e9-9fe2-b17d76074545.png)
Inconsistent.

![image](https://user-images.githubusercontent.com/4161918/58821818-3100c200-8636-11e9-905b-c23ee45444d9.png)
Not flagged...

_To be discussed:_ the value of this field for validation.

**Comment by DavorJ** [comment]
I also just checked the BAOL086X_179793:

![image](https://user-images.githubusercontent.com/4161918/58822346-92756080-8637-11e9-91f5-7e656a7f69fd.png)
INV=896, but they are not shown. This is because for these records, timestamps are missing. Plotting sequentially shows this:

![image](https://user-images.githubusercontent.com/4161918/58822260-66f27600-8637-11e9-9f97-a3014cbaea02.png)
Here also:

![image](https://user-images.githubusercontent.com/4161918/58822734-7a521100-8638-11e9-917a-1e1b3fec3562.png)
![image](https://user-images.githubusercontent.com/4161918/58822774-90f86800-8638-11e9-98c0-17d214220ff1.png)
All these red dots have no timestamp.

But not always as in the case of BAOL086X_179793: there some black points have no timestamp too!

_To be discussed_: INV is used with multiple meanings: no timestamp and as suspicious, but always inconsistent.

**Comment by fredericpiesschaert** [comment]
OK, so status code as it is seems of little value for validating the outlier procedure. We have two options:
- clean up some selected WATINA-series, but I think the available status-codes do not allow sufficient granularity to do this properly. We can't just introduce new status codes without compromising the WATINA-application
- check with the other partners if they have suitable validated data
this issue demonstrates the need for a controlled vocabulary

**Comment by fredericpiesschaert** [comment]
I think users also keep separate lists of 'suspicious data': (parts of) timeseries that should not be used for compensation, without being explicitely marked as invalid in the database. Is that correct @mathiaswackenier ?

**Comment by mathiaswackenier** [comment]
> I think users also keep separate lists of 'suspicious data': (parts of) timeseries that should not be used for compensation, without being explicitely marked as invalid in the database. Is that correct @mathiaswackenier ?

We do keep these lists, but they are not written down. The WATINA-application forces the user to visually check the timeseries and by doing so we can easily detect suspicious data. There is also a second way how we detect suspicious data and that is during the compensation and calibration. Outliers or suspicious data in the timeseries of the barometric sensors will cause mistakes on the timeseries that are easily visually detected. 

In short, the lists we have don't exist in hard-copy, but we are aware of which timeseries are unreliable.

**Comment by fredericpiesschaert** [comment]
I think the conclusion is that DRME_DMST_CDE is not useful for validating the algorithms
